pkg/refactor/logstore/dblogstore/mockdata/entlog_nestedLoop.json

```diff
@@ -0,0 +1,1928 @@
+          "invoker": "instance:c8a13535-b5ae-469b-98f2-e64a831067f9",
+          "loop-index": "0",
+          "namespace": "ns",
+          "namespace-id": "1c3c7f7b-c2c3-4f84-8377-eab3c7b8c11d",
+          "recipientType": "instance",
+          "state-id": "solve",
+          "state-type": "foreach",
+          "workflow": "looper",
+          "workflow-id": "db150cf6-df0e-4732-b978-a58ced5442f8"
+        }
+      },
+      {
+        "t": "2023-03-23T13:05:59.500639Z",
+        "level": "info",
+        "msg": "Child '6b9d3ee1-28a5-428a-880d-fa82cce03b5f' returned.",
+        "tags": {
+          "callpath": "/",
+          "instance-id": "c8a13535-b5ae-469b-98f2-e64a831067f9",
+          "invoker": "api",
+          "loop-index": "3",
+          "namespace": "ns",
+          "namespace-id": "1c3c7f7b-c2c3-4f84-8377-eab3c7b8c11d",
+          "recipientType": "instance",
+          "state-id": "solve",
+          "state-type": "foreach",
+          "workflow": "looperlooper",
+          "workflow-id": "ff4bf1cf-5533-4af2-961c-fd10e639950a"
+        }
+      },
+      {
+        "t": "2023-03-23T13:05:59.500651Z",
+        "level": "info",
+        "msg": "[3] Action returned. (4/4)",
+        "tags": {
+          "callpath": "/",
+          "instance-id": "c8a13535-b5ae-469b-98f2-e64a831067f9",
+          "invoker": "api",
+          "loop-index": "3",
+          "namespace": "ns",
+          "namespace-id": "1c3c7f7b-c2c3-4f84-8377-eab3c7b8c11d",
+          "recipientType": "instance",
+          "state-id": "solve",
+          "state-type": "foreach",
+          "workflow": "looperlooper",
+          "workflow-id": "ff4bf1cf-5533-4af2-961c-fd10e639950a"
+        }
+      },
+      {
+        "t": "2023-03-23T13:05:59.500689Z",
+        "level": "debug",
+        "msg": "Transforming state data.",
+        "tags": {
+          "callpath": "/",
+          "instance-id": "c8a13535-b5ae-469b-98f2-e64a831067f9",
+          "invoker": "api",
+          "loop-index": "3",
+          "namespace": "ns",
+          "namespace-id": "1c3c7f7b-c2c3-4f84-8377-eab3c7b8c11d",
+          "recipientType": "instance",
+          "state-id": "solve",
+          "state-type": "foreach",
+          "workflow": "looperlooper",
+          "workflow-id": "ff4bf1cf-5533-4af2-961c-fd10e639950a"
+        }
+      },
+      {
+        "t": "2023-03-23T13:05:59.501031Z",
+        "level": "info",
+        "msg": "Workflow looperlooper completed.",
+        "tags": {
+          "callpath": "/",
+          "instance-id": "c8a13535-b5ae-469b-98f2-e64a831067f9",
+          "invoker": "api",
+          "loop-index": "3",
+          "namespace": "ns",
+          "namespace-id": "1c3c7f7b-c2c3-4f84-8377-eab3c7b8c11d",
+          "recipientType": "instance",
+          "state-id": "solve",
+          "state-type": "foreach",
+          "workflow": "looperlooper",
+          "workflow-id": "ff4bf1cf-5533-4af2-961c-fd10e639950a"
+        }
+      }
+    ]
```

Comment:
Typically, test data should be analyzable by human eyes so that it is easy to write correct assertions. Your test file has 1928 which is really hard to human read.